Framework/Algorithms/inc/MantidAlgorithms/ExtractUnmaskedSpectra.h

```diff
@@ -0,0 +1,51 @@
+#ifndef MANTID_ALGORITHMS_EXTRACTUNMASKEDSPECTRA_H_
+#define MANTID_ALGORITHMS_EXTRACTUNMASKEDSPECTRA_H_
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAlgorithms/DllConfig.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** ExtractUnmaskedSpectra : TODO: DESCRIPTION
```

Comment:
I guess this TODO is a leftover from the class maker. Anyway the algorithm has its documentation page so it doesn't need any explanation here.